ArcGISRuntimeSDKQt_CppSamples/Layers/ExportVectorTiles/ExportVectorTiles.cpp

```diff
@@ -152,21 +152,16 @@ void ExportVectorTiles::startExport(double xSW, double ySW, double xNE, double y
       m_jobStatus = (int)s;
```

Comment:
```suggestion
      m_jobStatus = static_cast<int>(s);
```
I realize this isn't your code, but we should not be using C-style casts anywhere.